src/patternfly/components/Button/styles.scss

```diff
@@ -290,6 +312,66 @@
     }
   }
 
+  // Link and Icon Buttons
+  &.pf-m-link,
+  &.pf-m-icon {
+    background-color: transparent;
+    box-shadow: none;
+
+    &:hover,
+    &.pf-m-hover,
+    &:active,
+    &.pf-m-active,
+    &:focus,
+    &.pf-m-focus {
+      background-color: transparent;
+      box-shadow: none;
+    }
+  }
+
+  // Link Buttons
+  &.pf-m-link {
+    padding: var(--pf-c-button--m-link--PaddingTop) var(--pf-c-button--m-link--PaddingRight) var(--pf-c-button--m-link--PaddingBottom) var(--pf-c-button--m-link--PaddingLeft);
+    color: var(--pf-c-button--m-link--Color);
+
+    &:hover,
+    &.pf-m-hover {
+      color: var(--pf-c-button--m-link--hover--Color);
+    }
+
+    &:active,
+    &.pf-m-active {
+      color: var(--pf-c-button--m-link--active--Color);
+    }
+
+    &:focus,
+    &.pf-m-focus {
+      color: var(--pf-c-button--m-link--focus--Color);
+    }
+  }
+
+  // Icon Buttons
+  &.pf-m-icon {
+    padding: var(--pf-c-button--m-icon--PaddingTop) var(--pf-c-button--m-icon--PaddingRight) var(--pf-c-button--m-icon--PaddingBottom) var(--pf-c-button--m-icon--PaddingLeft);
+    line-height: var(--pf-c-button--m-icon--LineHeight);
```

Comment:
Can we add a comment before this line to tell people why we are changing the line height?